streaming/BUILD.bazel

```diff
@@ -70,10 +76,13 @@ cc_library(
         "src/util/*.h",
     ]),
     copts = COPTS,
-    strip_include_prefix = "src",
+    includes = [
+        "src", "src/util/"
```

Comment:
Why add "src/util/"? Shouldn't we include util as `#include "util/xxx"`? This has a more qualified path and can avoid many path conflict

Reply:
done